### PR TITLE
added metadata section

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ To collect System logs from the Citrix Cloud platform, you must have an authoriz
 
 The Citrix Cloud API integration retrieves system logs every 5 minutes.
 
+## Metadata Fields
+
+Metadata fields will be set, if the integration is configured with the SIEM forward option. See **Metadata Fields** table below:
+
+| Field Name | Value |
+| --- | --- |
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source.md:27]:
```suggestion
| :--- | :--- |
```

Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source.md:27]: Columns should be left-aligned as per the style guide.

**Review by JainM6 — CHANGES_REQUESTED** [review]
Looks good. Just a minor correction.

**Comment by JV0812** [comment]
Comments fixed.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source.md [hunk 2]
```diff
@@ -19,6 +19,14 @@ To collect System logs from the Citrix Cloud platform, you must have an authoriz
 
 The Citrix Cloud API integration retrieves system logs every 5 minutes.
 
+## Metadata Fields
+
+Metadata fields will be set, if the integration is configured with the SIEM forward option. See **Metadata Fields** table below:
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source.md:24]:
```suggestion
The following **Metadata Fields** will be set if you configure your integration with the SIEM forward option.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
@JV0812 For consistency with the rest of the C2C docs, this should be placed as a step in the [Set up Citrix Cloud Source section](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source/#set-up-citrix-cloud-source).

Here's an example from another C2C doc:
Screenshot 2023-04-13 at 11 40 22 AM

**Comment by JV0812** [comment]
In some of the c2c doc, Metadata fields are mentioned as a H2 section. [This app](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source/) has the exact `_siemparser` field and is a part of H2 and not the step.
Kindly suggest me which one to follow.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**